.gitignore

```diff
@@ -1,3 +1,7 @@
 Built
 *.aliases
 *.lvlps
+*.lvcodedb
+*.nivsprjcache
+*.nivssdfcache
+Source/Tests/System Tests/Scalar Channels Testing/Assets/VS FPGA Addon Scalar Testing/.cache/
```

Comment:
Replace this line with **.cache/** , what we want here is a blanket-exclude of all .cache folders, since more will be coming down the line, when adding tests.